src/news.ts

```diff
@@ -24,7 +24,8 @@ export class LodestoneNews {
   protected region: Region
   protected count: number
   protected request?: RequestOptions
-
+  protected before?: Date
+  protected after?: Date
```

Comment:
```suggestion

```

no need to be class fields

Reply:
`before`、`after`作为一个可选项，应与`request`保持一致

Reply:
Well, then the class-invoking style should be:

```ts
const lodestone = new LodestoneNews({ before: xxx, after: yyy })
// ...
lodestone.getNews() // get news between xxx and yyy *implicitly*
lodestone.getNews({ after: xxx }) // get nothing QAQ
```

It is weird to call a function without arguments but get data that filtered by some invisible mechanics.
So I insist to add `before` and `after` on `getNews` function, instead of being class fields.